openapi_python_client/templates/endpoint_macros.py.jinja

```diff
@@ -37,7 +37,7 @@ params: Dict[str, Any] = {
             {% if property.required and not property.nullable %}
 params.update({{ property_name }})
             {% else %}
-if {% if not property.required %}not isinstance({{ property_name }}, Unset){% endif %}{% if not property.required and property.nullable %} and {% endif %}{% if property.nullable %}{{ property_name }} is not None{% endif %}:
+if not isinstance({{ property_name }}, Unset) and {{ property_name }} is not None:
```

Comment:
I think this was a mistake since now we get `.update()` twice in the generated code. Also mypy probably will get mad about non-overlapping type checks or whatever.

Reply:
Oops, yeah, probably a mistake from my merge. I think the two `.update()`s you're referring to are actually updating different things, though.